scripts/rimpl.ts

```diff
@@ -0,0 +1,72 @@
+import {parseProcessArgument} from '../src/util/args';
+import {openYamlFileAsObject, saveYamlFileAs} from '../src/util/yaml';
+import {Observatory} from '../src/util/observatory';
+import {ModelsUniverse} from '../src/util/models-universe';
+
+/**
+ * For each graph builds params, then passes it to computing fn.
+ */
+const calculateImpactsBasedOnGraph =
+  (impl: any, modelsHandbook: ModelsUniverse) => async (service: string) => {
+    const serviceData = impl.graph.children[service];
+    const {pipeline, observations, config} = serviceData;
+
+    const observatory = new Observatory(observations);
+
+    for (const model of pipeline) {
+      const instance: any = await modelsHandbook.initalizedModels[model](
```

Comment:
Took me a while to figure out what the variables names meant, just suggesting alternatives that are more descriptive

```suggestion
      const modelInstance: any = await modelsHandbook.initalizedModels[modelName](
```